ui-v2/app/templates/components/healthchecked-resource.hbs

```diff
@@ -1,32 +1,38 @@
-    <header class={{if service 'with-service' }}>
-      <strong>{{address}}</strong>
-        <a href={{href}}>
-            <span>{{name}}</span>
-            <em>{{service}}</em>
-        </a>
-    </header>
-    {{! its important to keep this <ul> with no whitespace so we can use :empty in css }}
-    <ul>{{#if status }}
-        <li class={{status}}>
-            <a href={{href}}>
-                <strong>{{status}}</strong>
-            </a>
-        </li>
-{{ else }}
-    {{#each unhealthy as |check| }}
-        <li class={{check.Status}}>
-            <a href={{href}}>
-                <strong>{{ check.Status }}</strong>
-                <span>{{ check.Name }}</span>
-            </a>
-        </li>
-    {{/each}}
-    {{#if (gt healthy.length 0)}}
-        <li class="passing">
-            <a href={{href}}>
-                <strong>{{healthy.length}}</strong>
-                <span>other passing {{pluralize healthy.length 'check' without-count=true}}</span>
-            </a>
-        </li>
+{{#stats-card}}
+  {{#block-slot 'icon'}}{{#if false}}<span data-tooltip="Leader">Leader</span>{{/if}}{{/block-slot}}
```

Comment:
P.S. This also adds a visible marker for being able to show which node is the leader. This is currently purposefully set to `false` until its 100% clear how we are going to get the information required for this from the API.

See:

https://github.com/hashicorp/consul/issues/5073
https://github.com/hashicorp/consul/pull/5455

https://www.consul.io/api/status.html#get-raft-leader